pages/community/index.js

```diff
@@ -110,225 +47,149 @@ function CommunityIndexPage() {
         </div>
       </div>
-      <div className="z-40 mt-[30rem] md:mt-10 bg-white w-full md:h-130 rounded-lg shadow-xl md:flex md:justify-between">
-        <div className="p-10 md:flex justify-between md:w-2/5 md:h-full flex-col text-center md:text-left">
+      <div className="md:flex md:justify-between w-full mt-20">
+        <div className="md:w-[45%] flex flex-col justify-between mt-10 md:mt-0">
           <div>
-            <Heading
-              level="h2"
-              typeStyle="heading-md"
-              textColor="text-purple-300"
-            >
-              All community info, tracked
-            </Heading>
+            <Card
+              type="small"
+              tagline="Community"
+              icon="🚀"
+              heading="Community Goals 2023"
+              description="Look into the AsyncAPI community building/maintenance goals for
+              2023 and help us improve."
+              link="https://github.com/asyncapi/community/discussions/513"
+            />
           </div>
-          <div>
-            <Heading
-              level="h2"
-              typeStyle="heading-lg"
-              className="mt-10 md:mt-0"
-            >
-              AsyncAPI Slack
-            </Heading>
-            <Heading
-              level="h2"
-              typeStyle="body-lg"
-              textColor="text-gray-700"
-              className="text-slate-500 text-sm mt-10"
-            >
-              AsyncAPI’s incredible community of developers, designers,
-              technical writers, and more hail from over 83 countries. We
-              actively contribute, collaborate, and mentor others on how to
-              build with AsyncAPI.
-            </Heading>
-            <div className="mt-10">
-              <Button
-                text="Join AsyncAPI slack"
-                buttonSize="medium"
-                href="https://asyncapi.com/slack-invite"
-                target="_blank"
-              />
-            </div>
+          <div className="mt-10 md:mt-0">
+            <Card
+              type="small"
+              taglineBg=""
+              tagline="Rockstars"
+              icon="🏅"
+              heading="Recognize AsyncAPI OSS contributors"
+              description="Recognize the people who inspire and contribute to our OSS
+                    project and community."
+              bg="bg-primary-200"
+              link="https://github.com/orgs/asyncapi/discussions/593"
+            />
           </div>
         </div>
-        <div className="w-full h-half-screen hidden md:block md:h-full md:w-3/6 md:flex md:justify-end bg-channelCover md:bg-left bg-cover"></div>
+        <div className="md:w-[50%] mt-10 md:mt-0">
+          <Card
+            taglineBg="bg-primary-200"
+            heading="Meet Folks that are Redefining the Initiative"
+            description="Learn how to become a technical steering committee member and see our current tsc members."
+            tagline="Projection"
```

Comment:
```suggestion
            tagline="TSC"
```

Yeah... Projection doesn't really say anything to me here, I think we should be clear and direct.